File: 08.C# - Quality code/HW11/HW/Unit-Testing/SchoolSystem/School.cs
using System;

public class School
{
    private string name;

    public School(string name)
    {
        this.Name = name;
    }

    public string Name
    {
        get
        {
            return this.name;
        }

        set
        {
            if (value == null)
            {
                throw new ArgumentNullException("The school name is not allowed to be null!");
            }

            this.name = value;
        }
    }
}

File: 10.JS3/Perfect HW1/02. SchoolRepository - Classical/scripts/School.js
/// <reference path="OOPUtils.js" />
/// <reference path="ClassOfStudents.js" />

(function () {
    School = Class.create({
        init: function (name, town) {
            this.name = name;
            this.town = town;
            this.classesOfstudents = [];
        },
        addClassOfStudents: function (classOfStudents) {
            this.classesOfstudents.push(classOfStudents);
        }
    });
}())